pages/badges/badge-single.jsx

```diff
@@ -234,16 +234,26 @@ var BadgePage = React.createClass({
     );
   },
 
+  getShareCodes: function() {
+    var url = encodeURIComponent(window.location.toString());
+    var msg = encodeURIComponent(`I earned the Mozilla ${this.state.badge.title} badge`);
```

Comment:
@Pomax - are you suggesting the page with details about the particular badge, or a page that shows what badges the user has earned?